devTools/docker/sync-s3-images.sh

```diff
@@ -13,12 +13,12 @@ fi
 # for local development, it should be owid-image-upload/local-yourname
 # at least until we decide to instead host images locally, if ever
 
-if ! grep -q 'profile owid-spaces' ~/.aws/config; then
-  echo 'Please configure your S3 credentials for profile owid-spaces:'
+if ! grep -q 'profile owid-r2' ~/.aws/config; then
+  echo 'Please configure your S3 credentials for profile owid-r2:'
   echo
-  echo '  aws configure --profile=owid-spaces'
+  echo '  aws configure --profile=owid-r2'
   echo
   exit 1
 fi
 
-aws --profile=owid-spaces --endpoint=https://nyc3.digitaloceanspaces.com s3 sync s3://owid-image-upload/production/ s3://$IMAGE_HOSTING_BUCKET_PATH/ --acl public-read
\ No newline at end of file
+aws --profile=owid-r2 --endpoint=https://078fcdfed9955087315dd86792e71a7e.r2.cloudflarestorage.com s3 sync s3://owid-image-upload/production/ s3://$IMAGE_HOSTING_BUCKET_PATH/ --acl public-read
```

Comment:
Based on our discussion, and tool preference, we should use rclone instead, and guide on how to set up r2 credentials in rclone.

We should assume a multi-user setup, which means our config for rclone should use a profile called `owid-r2` rather than `r2`.